domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/gtfs/Agency.java

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright (c) 2020. MobilityData IO.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.domain.entity.gtfs;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.base.Notice;
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.error.MissingRequiredValueNotice;
+
+import java.util.List;
+
+/**
+ * Class for all entities defined in agency.txt. Can not be directly instantiated: user must use the
+ * {@link AgencyBuilder} to create this.
+ */
+@SuppressWarnings("ALL")
```

Comment:
Seems intense. At least add a comment to explain why

Reply:
In fact, I suggest to remove unused methods that provoke warnings (they're all getters). They will be implemented when needed (when writing code to implement semantic rules).